inferentia/scripts/setup.sh

```diff
@@ -106,36 +127,32 @@ rm -rf build && mkdir build && cd build
 cmake -DTRITON_ENABLE_GPU=ON -DCMAKE_INSTALL_PREFIX:PATH=`pwd`/install ..
 make triton-python-backend-stub -j16
 
-# Install Neuron Pytorch
-# https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-intro/install-pytorch.html
-cd ${INFRENTIA_PATH}
-. /etc/os-release
-tee /etc/apt/sources.list.d/neuron.list > /dev/null <<EOF
-deb https://apt.repos.neuron.amazonaws.com ${VERSION_CODENAME} main
-EOF
-wget -qO - https://apt.repos.neuron.amazonaws.com/GPG-PUB-KEY-AMAZON-AWS-NEURON.PUB |  apt-key add -
-apt-get update && \ 
-    apt-get install -y       \
-        linux-headers-$(uname -r) \
-        aws-neuron-dkms           \
-        aws-neuron-runtime-base   \
-        aws-neuron-runtime        \
-        aws-neuron-tools 
-export PATH=/opt/aws/neuron/bin:$PATH
-
-# Install Neuron-pytorch (neuron-cc)
+# Set Pip repository  to point to the Neuron repository
+# since we need to use pip to update: 
+#  https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
+pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
 conda config --env --add channels https://conda.repos.neuron.amazonaws.com
-conda install torch-neuron -y
 
-# Upgrade torch-neuron and install transformers
-# need to use pip to update: 
-# https://aws.amazon.com/blogs/developer/neuron-conda-packages-eol/
-pip config set global.extra-index-url https://pip.repos.neuron.amazonaws.com
-pip install --upgrade torch-neuron torchvision "transformers==4.6.0"
+if [ $USE_TENSORFLOW -eq 0 ]
+then
+    conda install tensorflow-neuron pillow -y
+    #Update Neuron TensorFlow
+    pip install --upgrade tensorflow-neuron==1.15.5.* neuron-cc
+    # Update Neuron TensorBoard
+    pip install --upgrade tensorboard-plugin-neuron
+fi
+
+if [ $USE_PYTORCH -eq 0 ]
+then
+    conda install torch-neuron torchvision -y
```

Comment:
I guess these dependency comes from the demos from AWS Neuron SDK? There might be some more dependencies that a user might need to successfully deploy their models. 
Just something to keep in mind for future.

Reply:
same as above. I think it might be something necessary to run the examples. However these are the minimum set of instructions for installation in their instructions. What else do you think the user would need? I can add them now

Reply:
I don't think we might tell that right now. It depends really on their model right. I was wondering for making them user configurable in the script. But I guess for now let's just focus on mlperf models. Take a note of this however for future.